card/mod/follow/spec/set/all/notify_spec.rb

```diff
@@ -136,6 +136,17 @@ def notification_email_for card_name, followed_set: "#{card_name}+*self"
   end
 
   describe "#notify_followers" do
+    # Normally, delayed events renew the cache, which entails clearing the local cache.
+    # That breaks these tests, because if the local cache is cleared, then the +*account
+    # card that receives the :send_change_notice method is a _different_ object than
+    # the one expecting it.
+    #
+    # An alternative approach would be to insert _only_ the account object into the
+    # cache when the delayed job is started.  This would more reliably test that the
+    # delayed job can get everything it needs with a clean cache.
```

Comment:
This may be why we changed the renew call in the first place.

I'm even more confident now that #renew is right, because I can see how it would break this spec but not because it actually broke the functionality.  (In fact I tested and it was working correctly, but the spec couldn't tell).